setup/zpush.sh

```diff
@@ -41,18 +41,18 @@ if [ $needs_update == 1 ]; then
 	mv /tmp/z-push/*/src /usr/local/lib/z-push
 	rm -rf /tmp/z-push.zip /tmp/z-push
 
-	# Create admin and top scripts with PHP_VER  
-	rm -f /usr/sbin/z-push-{admin,top}
-    echo '#!/bin/bash' > /usr/sbin/z-push-admin
-    echo php"$PHP_VER" /usr/local/lib/z-push/z-push-admin.php '"$@"' >> /usr/sbin/z-push-admin
-    chmod 755 /usr/sbin/z-push-admin
-    echo '#!/bin/bash' > /usr/sbin/z-push-top
-    echo php"$PHP_VER" /usr/local/lib/z-push/z-push-top.php '"$@"' >> /usr/sbin/z-push-top
-    chmod 755 /usr/sbin/z-push-top
-	
 	echo $VERSION > /usr/local/lib/z-push/version
 fi
 
+# Create admin and top scripts with PHP_VER  
```

Comment:
Curious here, why did you move this code out of the if-else block?

Reply:
As @matidau noted in #2431, this code updates the used php version of zpush, but the code is only run on an update of zpush. Because PHP_VER is changed, the scripts should be updated even without a zpush update.